pyqtorch/noise.py

```diff
@@ -98,6 +86,30 @@ def to(self, device: torch.device) -> Noise:
         self._device = device
         return self
 
+    def tensor(self, values: dict[str, Tensor] = {}, n_qubits: int = 1) -> list[Tensor]:
+        blockmats = self._tensor(values)
+        mats = []
+        for blockmat in blockmats:
+            if n_qubits == 1:
+                mats.append(blockmat)
+            else:
+                full_sup = tuple(i for i in range(n_qubits))
+                support = tuple(sorted(self.qubit_support))
+                mat = (
+                    IMAT.clone().to(self.device).unsqueeze(2)
```

Comment:
Probably needs to add the dtype for IMAT, to handle complex64 and complex128 like all quantum ops.